src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,35 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type PostInfoProp = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<PostInfoProp> = ({ post }) => {
+  const { user } = post;
```

Comment:
Let's make a destructuring from post
```suggestion
  const { 
    user,
    ...anotherVariables
   } = post;
```